### ISSUE TITLE
[PackageId]: [version(s)]

### ISSUE THREAD

**Issue by stijnnel** [dotnet/aspnetcore#50736]
### Package Id

Microsoft.AspNetCore.WebUtilities

### Package Version(s)

2.2.0

### Additional details

why is 2.2.0 deprecated and 2.1.1 and version 2.1.0 not?  
couple of months only older versions than 2.1.0 or previews were marked deprecated.

**Comment by martincostello** [comment]
ASP.NET Core 2.2 reached end-of-life before 2.1, and there's also some special-casing for support for 2.1, which has also reached end of life, when it is running on .NET Framework.

These packages are now part of the shared framework since ASP.NET Core 3.0, which are made available via a FrameworkReference rather than a PackageReference. This is why the packages are marked as deprecated and why there are no newer versions.

You can find more details about this here: https://learn.microsoft.com/aspnet/core/fundamentals/target-aspnetcore#use-the-aspnet-core-shared-framework

In short, for new application development you should not reference these NuGet packages and use a framework reference instead.
